docs/eigenda/deployed-contracts.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 8
+title: Contracts
+---
+
+## EigenDA Contracts
+
+For more information on the contracts and their technical specifications, please see:
+[EigenLayer Middleware Docs](https://github.com/Layr-Labs/eigenlayer-middleware/tree/m2-mainnet/docs).
+
+### Current Mainnet Deployment
+
+No contracts have been deployed to mainnet yet.
+
+### Current Testnet Deployment
+
+The current testnet deployment of EigenDA is [`v0.3.0`](https://github.com/Layr-Labs/eigenda/releases/tag/v0.3.0) part of which includes middleware contracts at version [`v0.1.1-goerli-m2`](https://github.com/Layr-Labs/eigenlayer-middleware/tree/testnet) release. Below are the deployed contract addresses and links to source.
```

Comment:
"version [v0.1.1-goerli-m2](https://github.com/Layr-Labs/eigenlayer-middleware/tree/testnet) release" wording is a little bit awkward, but not blocking